docs/using.rst

```diff
@@ -0,0 +1,58 @@
+==============================
+Using the Let's Encrypt client
+==============================
+
+Prerequisites
+=============
+
+The demo code is supported and known to work on **Ubuntu only** (even
+closely related [Debian is known to fail]
+(https://github.com/letsencrypt/lets-encrypt-preview/issues/68)).
+
+Therefore, prerequisites for other platforms listed below are provided
+mainly for the [developers](#hacking) reference.
+
+In general:
+
+* `swig`_ is required for compiling `m2crypto`_
+* `augeas`_ is required for the `python-augeas` bindings
+
+Ubuntu
+------
+
+::
+
+    sudo apt-get install python python-setuptools python-virtualenv python-dev \
+                 gcc swig dialog libaugeas0 libssl-dev ca-certificates
+
+
+Mac OSX
+-------
+
+::
```

Comment:
It doesn't render correctly in GitHub. I believe you're missing an empty line here.